containers/ws/atomic-install

```diff
@@ -42,6 +42,9 @@ mkdir -p /host/var/lib/cockpit
 chmod 775 /host/var/lib/cockpit
 chown root:wheel /host/var/lib/cockpit
 
+# For sharing ssh's known hosts with container
+mkdir -p /etc/ssh
```

Comment:
Unfortunately we currently have to maintain this file in two places. Can you also open a PR with this change here.

https://github.com/cockpit-project/cockpit-container